frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -63,50 +70,67 @@ export class TemplateValidations {
     if (this === otherTempValidations) {
       return true;
     }
+    const isBusesEqual = (buses, otherBuses) =>
+      buses.size === otherBuses.size && [...buses].every((bus) => otherBuses.has(bus));
 
     // Check if two sets of bus validations are the same - if the allowed and recommended buses are the same
-    const allowedBuses = this.getAllowedBuses();
-    const recommendedBuses = this.getRecommendedBuses();
-    const otherAllowedBuses = otherTempValidations.getAllowedBuses();
-    const otherRecommendedBuses = otherTempValidations.getRecommendedBuses();
-
-    return (
-      allowedBuses.size === otherAllowedBuses.size &&
-      recommendedBuses.size === otherRecommendedBuses.size &&
-      [...allowedBuses].every((bus) => otherAllowedBuses.has(bus)) &&
-      [...recommendedBuses].every((bus) => otherRecommendedBuses.has(bus))
-    );
+    const allowedBuses = this.getAllowedBuses(DiskType.DISK);
```

Comment:
We could also do `[DiskType.DISK, DiskType.CDROM].every` but I do not mind either way